dt: 0.1
dv: 0.01
dyaw: 0.01
max_speed: 0.42
min_speed: 0.0
max_accel: 5.0
max_yawrate: 1.00
max_dyawrate: 2.0
robot_radius: 0.17
predict_time: 2.0
l_ob_cost_gain: 1.0
speed_cost_gain: 2.7
omega_cost_gain: 0.0
to_g_goal_cost_gain: 0.0
dis_g_goal_cost_gain: 0.0
to_g_path_cost_gain: 5.0
dis_g_path_cost_gain: 5.0
